analysis/geospatial_check.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS geospatial_check_table;
+CREATE TABLE IF NOT EXISTS geospatial_check_table(
+    v character varying,
+    result character varying
+);
+
+DELETE FROM geospatial_check_table
+WHERE v = :'ccp_v';
+
+INSERT INTO geospatial_check_table (
+select :'ccp_v' as v, 	 
+        jsonb_agg(t) as result
+from (
+    select jsonb_agg(json_build_object('projectid', tmp.projectid, 'magencyacr',tmp.magencyacr, 
```

Comment:
there is a typo here with `agencyacr` should be `agencyacro`

Reply:
yes. Also seeing a unrelated error in the `analysis/agency_validated_geoms_summary_table.sql`

Reply:
based on what I have pulling down, the column name is magencyacr

Reply:
let me double check